airflow/providers/slack/transfers/sql_to_slack.py

```diff
@@ -33,6 +33,10 @@
     from airflow.utils.context import Context
 
 
+class SqlToSlackNullOutputException(AirflowException):
+    """An exception that indicates the sql output is empty and hence should not be sent with slack message."""
```

Comment:
Do we need separate exception for that? Or we just could use ValueError instead?

Reply:
I think so, it's not necessarily a parameter input. But also, based on comment below, won't need this Exception, just skipping instead of failing is the desired behavior.